### PR TITLE
Add getId function to python API

### PR DESCRIPTION
**Pull request by alex-ozdemir**
(Z3 exposes it to facilitate custom hashes)

### CODE REVIEW

**Comment by makaimann** [comment]
Thanks for the PR! It looks like `getId` is already defined here:
https://github.com/cvc5/cvc5/blob/87ddf639068e9bb917c6c00c6f2c73ec6844af9a/src/api/python/cvc5.pxi#L1487

And tested here:
https://github.com/cvc5/cvc5/blob/b016f603f2a3709faaf90008a09f2567d7283ff3/test/python/unit/api/test_term.py#L39

Do you still want to add this example though?

**Comment by alex-ozdemir** [comment]
Oh, whoops! I seemed to have missed this PR: https://github.com/cvc5/cvc5/pull/6477.

Closing.

**Comment by alex-ozdemir** [comment]
Whoops again, yeah, me might as well at the example.

File: examples/api/python/id.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# #############################################################################
+#
+# A simple demonstration of the solving capabilities of the cvc5 sets solver
```
Comment (makaimann) [inline examples/api/python/id.py:14]: This description should be updated for this example.

File: examples/api/python/id.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+# through the Python API. This is a direct translation of sets-new.cpp.
+##
+
+import pycvc5
+from pycvc5 import kinds
+
+if __name__ == "__main__":
+    slv = pycvc5.Solver()
+
+    # Optionally, set the logic. We need at least UF for equality predicate,
```
Comment (makaimann) [inline examples/api/python/id.py:24]: Similarly, this comment is a little confusing because the logic for this example is arbitrary, right?

**Review by makaimann — CHANGES_REQUESTED** [review]
Sounds good! Left a few easy-to-address change requests.

File: examples/api/python/id.py (hunk 2 above)
Comment (alex-ozdemir) [reply examples/api/python/id.py]: Good catch. Fixed in c7a99b06dd.

File: examples/api/python/id.py (hunk 1 above)
Comment (alex-ozdemir) [reply examples/api/python/id.py]: Good catch. Fixed in c7a99b06dd.

**Review by makaimann — APPROVED** [review]
LGTM, thanks for the changes!

**Pull request merged**